### ISSUE TITLE
rip + rvm: Document Workflow and Interaction

### ISSUE THREAD

**Issue by rue** [defunkt/rip#100]
Sort of a placeholder issue unless this has already been discussed in detail (librelist is being difficult.) There is some obvious overlap, such as rvm's gemsets, and probably less obvious cases. Should try to document at least the most common cases or best practices of usage alongside, on top of etc. rvm.

**Comment by josh** [comment]
I believe rip's env are a replacement rvm's gemsets. I don't think you would use both.

But we still need to work out rip's multiple ruby version support in general.

**Comment by defunkt** [comment]
Yeah, I'd definitely like to get rip working with rvm as more and more people adopt it.

**Comment by pboling** [comment]
I just (re)found rip, thanks to [homesick](https://github.com/technicalpickles/homesick).  Is rip dead?  I'd like to be able to have the gem caching of rip integrated with rvm to mitigate all the re-downloading of gems for each gemset.

**Comment by amerine** [comment]
@pboling Dead.
